src/OpenLoco/src/Input/Keyboard.cpp

```diff
@@ -342,6 +342,16 @@ namespace OpenLoco::Input
                 }
             }
 
+            ti = WindowManager::find(WindowType::objectSelection);
+            if (ti != nullptr)
```

Comment:
I think we should start handling these through the window manager, ideally passing through to an event for the active window only. Something for a future PR though.

Reply:
Yeah i was a bit confused as to what this was about.